apps/website/src/content/docs/tree.mdx

```diff
@@ -1,27 +1,149 @@
 ---
 title: Tree
-description: A tree provides a list of data.
+description: A tree provides a hierarchial lists of data with nested expandable levels.
 thumbnail: Tree
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='tree--basic' />
+## Usage
+
+A tree can be used to organize data in an application specific way, or it can be used to sort, filter, group, or search data as the user deems appropriate. To initialize the tree component, the following props are required:
```

Comment:
The first sentence should be moved out of the "Usage" heading because it has nothing do with usage.

Reply:
I have moved this out of "Usage".